core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -474,8 +474,6 @@ describe('Exploration editor page component', function() {
         ' function on index 1 of ctrl.EDITOR_TUTORIAL_OPTIONS array',
       () => {
         var element = angular.element('div');
-        // @ts-ignore is being used in order to ignore JQuery properties that
```

Comment:
Just checking -- what changed? Why was it needed before and not now?

Reply:
`window` was missing a definition for `$` added that.